drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -58,6 +58,16 @@ class DragAndDropBlock(XBlock, XBlockWithSettingsMixin, ThemableXBlockMixin):
         default=STANDARD_MODE
     )
 
+    max_attempts = Integer(
+        display_name=_("Maximum number of attempts"),
+        help=_(
+            "Limits the number of attempts learner can use to complete the problem. "
+            "For unlimited attempts, set to zero. This setting have no effect in standard mode."
```

Comment:
Actually, this field and related studio edit functionality have already been implemented in https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/81, you should probably remove this code from this PR.